website/docs/r/data_protection_backup_vault.html.markdown

```diff
@@ -55,7 +55,9 @@ The following arguments are supported:
 
 -> **Note:** The `retention_duration_in_days` is the number of days for which deleted data is retained before being permanently deleted. Retention period till 14 days are free of cost, however, retention beyond 14 days may incur additional charges. The `retention_duration_in_days` is required when the `soft_delete` is set to `On`.
 
-* `soft_delete` - (Optional) The state of soft delete for this Backup Vault. Possible values are `AlwaysOn`, `Off` and `On`. Defaults to `On`.
+* `immutability` - (Optional) The state of immutability for this Backup Vault. Possible values are `Disabled`, `Locked`, and `Unlocked`. Defaults to `Disabled`. Changing this from `Locked` to anything else forces a new Backup Vault to be created.
```

Comment:
This is more for if we want the naming to be better but I think `Unlocked` isn't very clear that it's actually enabling immutability. It'd be a bit more work on our end but what are your thoughts on changing `Unlocked` to `Enabled`?

Reply:
I think its best if we just keep it matching the API values even if the portal uses something else here despite it being confusing